README.md

```diff
@@ -153,76 +153,135 @@ On certain pages an API endpoint is called to receive experiment analysis data f
+
+#### make up_db
 
 Start postgresql, redis, autograph, kinto on their respective ports to allow running the Django runserver and yarn watchers locally (non containerized)
 
-### up_django
+#### up_django
 
 Start Django runserver, Celery worker, postgresql, redis, autograph, kinto on their respective ports to allow running the yarn watchers locally (non containerized)
 
-### up_detached
+#### up_detached
 
-Start all containers in the background (not attached to shell)
+Start all containers in the background (not attached to shell).  They can be stopped using `make kill`.
 
-### check
+### Running tests and checks
 
-Run all test and lint suites, this is run in CI on all PRs and deploys
+#### make check
 
-### py_test
+Run all test and lint suites, this is run in CI on all PRs and deploys.
 
-Run python tests
+#### make py_test
 
-### migrate
+Run only the python test suite.
 
-Apply all django migrations
+#### make bash
 
-### load_locales_countries
+Start a bash shell inside the container.  This lets you interact with the containerized filesystem and run Django management commands.
 
-Populates locales and countries
+##### Helpful Python Tips
+You can run the entire python test suite without coverage using the Django test runner:
 
-### load_dummy_experiments
+        ./manage.py test
 
-Populates db with dummy experiments
+For faster performance you can run all tests in parallel:
 
-### bash
+        ./manage.py test --parallel
 
-Start a bash shell inside the container (this lets you interact with the containerized filesystem and run Django management commands)
+You can run only the tests in a certain module by specifying its Python import path:
 
-### ssl
+        ./manage.py test experimenter.experiments.tests.api.v5.test_serializers
 
-Create dummy SSL certs to use the dev server over a locally secure
-connection. This helps test client behaviour with a secure
-connection. This task is run automatically when needed.
+For more details on running Django tests refer to the [Django test documentation](https://docs.djangoproject.com/en/3.1/topics/testing/overview/#running-tests)
 
-### kill
+To debug a test, you can use ipdb by placing this snippet anywhere in your code, such as within a test method or inside some application logic:
+
+        import ipdb
+        ipdb.set_trace()
+
+Then invoke the test using its full path:
+
+        ./manage.py experimenter.some_module.tests.some_test_file.SomeTestClass.test_some_thing
+
+And you will enter an interactive iPython shell at the point where you placed the ipdb snippet, allowing you to introspect variables and call methods
+
+For coverage you can use pytest, which will run all the python tests and track their coverage, but it is slower than using the Django test runner:
+
+        pytest --cov --cov-report term-missing
+
+You can also enter a Python shell to import and interact with code directly, for example:
+
+        ./manage.py shell
```

Comment:
Yo dawg, heard you liked shells, so we put a shell inside your shell

Reply:
![4c81da5ad2758d01845815f51a35a196](https://user-images.githubusercontent.com/119884/113200236-113c8400-9236-11eb-9865-c8a21c6ab2ba.gif)